src/app/auth/auth.service.ts

```diff
@@ -77,6 +88,11 @@ export class AuthService {
     // do clear user cache here
   }
 
+  private _loginAPILogin(body: LoginRequParams) {
+    body.from = 'App';
+    return this.request.loginAPIPost(api.loginAPI.login, body);
```

Comment:
@sasangachathumal  why is the different between writing this as 
`return this.request.post(api.loginAPI.login + 'login', body);` 

compared to this code in line 93?

Could you please explain why is it necessary to clone the [entire post function](https://github.com/intersective/app-v2/blob/AV2-924/update-login-api-call/src/app/shared/request/request.service.ts#L142-L165)?

Reply:
we still need both logins. when we login using login API, it return apikey right? but we can’t use it to call core APIs. we need to call core API login with that apikey to get apikey for login. that’s what i ask from shown in standup yesterday. that why i can’t just simply replace our core login with login api. i need to add new method to call login API.

it same like if user coming to appv2 from global login, url contain api key, in global login method we call core api login with that apikey to get correct one.

login API return an global apikey, but to call stack apis we need to get api key from that stack core api.

Reply:
and also it, it's calling login API `login` endpoint.
`return this.request.post(api.loginAPI.loginlogin, body);`
not this
`return this.request.post(api.loginAPI.login + 'login', body);`

Reply:
you can see api const values.
https://github.com/intersective/app-v2/pull/1207/files?file-filters%5B%5D=.ts#diff-0cc1fef6a75cb019a3dd68c49beec2725b302d53b51a98321f908b5da6cbe518R25-R28